Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM ubuntu:18.04
-RUN apt-get update
-RUN apt-get install -y \
+
+RUN apt update
```

Comment:
From `man apt`:

```
SCRIPT USAGE AND DIFFERENCES FROM OTHER APT TOOLS
       The apt(8) commandline is designed as an end-user tool and it may change behavior between versions. While it tries not to break backward compatibility this is not guaranteed either if a change seems
       beneficial for interactive use.

       All features of apt(8) are available in dedicated APT tools like apt-get(8) and apt-cache(8) as well.  apt(8) just changes the default value of some options (see apt.conf(5) and specifically the
       Binary scope). So you should prefer using these commands (potentially with some additional options enabled) in your scripts as they keep backward compatibility as much as possible.
```

Reply:
Good point, I removed the commit from the pull request